### DISCUSSION TITLE
Repeater block with default list

### DISCUSSION THREAD

**Discussion by Nexulo** [area17/twill#1665] [Ideas]
Hello together

I do not know if this function already exists. But it would be very useful to be able to specify a default list for a repeater.

For example, I have an enumeration, which has mostly the same inputs but still must be able to be extended individually.

**Comment by haringsrob** [comment]
Hey @maisen20,

I think what you could do is on create, and generate a repeater for your model for each of the enumeration items? 

A bit like a database seeder but from within your repository or model?

Hope I understood your question correctly.
